src/board.py

```diff
@@ -0,0 +1,20 @@
+class Board:
+    border = "⊱ –––––– {⋆⌘⋆} –––––– ⊰",
+    divider = "      ---+---+---"
```

Comment:
Should the board know how to render itself for the cli? 
what if we were to use LED lights on a raspberry PI?
Is the board only doing one thing?

Reply:
I felt that the board should know how to turn itself into a string that the cli could display, maybe I need to adjust my thought process?